packages/slate-react/src/components/slate.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useMemo } from 'react'
+import { Editor, Node, Operation, Range } from 'slate'
```

Comment:
moving this to a different module was a good decision i think, it took a little bit to find it just diving through github.